### DISCUSSION TITLE
QBtn href attribute does nothing in CDN install

### DISCUSSION THREAD

**Discussion by jonmor51** [quasarframework/quasar#12345] [General - Components / Directives / etc] [answered]
I wonder whether I am misunderstanding something here. I'm testing out Quasar in UMD/Standalone mode. I have a QBtn (otherwise working fine). I have added an 'href' attribute, assuming that clicking on the button would cause a page navigation to the URL but it does nothing. Nothing shows up in the dev console, and nothing happens. In fact even when I use this example from the docs `<q-btn href="start/pick-quasar-flavour" label="With href" push color="purple" />` it does nothing. This is with Quasar 2.3.1. What am I missing?

**Comment by jonmor51** [comment, marked as answer]
Ah, simple answer! Must have been a recent introduction, as changing to Quasar 2.5.3 solves the problem!

**Reply by rstoenescu** [reply]
Yes, it's been introduced recently.
